### PR TITLE
Try a new build for 0.13.1

### PR DESCRIPTION
**Pull request by blink1073**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by blink1073** [comment]
@conda-forge-admin, please rerender

**Comment by blink1073** [comment]
@conda-forge/core, something seems to be amiss with the CircleCI configuration.  We recently switched from `noarch` to `py3k` only.

**Comment by CJ-Wright** [comment]
Yep we're working on it.

**Comment by blink1073** [comment]
All green, good to merge?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,9 +10,9 @@ source:
   sha256: f880eada0b8b1f524d5951dc6fcae0d13b169897fc8a247d75fb5beadd69c5f0
 
 build:
-    number: 1
+    number: 2
     skip: True  # [py2k]
```
[the remark this answers is not in the document]
Comment (blink1073) [reply recipe/meta.yaml]: Thanks!  Let's give that a shot...

**Comment by blink1073** [comment]
@conda-forge-admin, please rerender

**Comment by blink1073** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ requirements:
     - setuptools
 
   run:
-    - python
+    - python >=3.5
```
Comment (mbargull) [inline recipe/meta.yaml]: You should also add that constraint to `requirements/host`.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -10,9 +10,9 @@ source:
   sha256: f880eada0b8b1f524d5951dc6fcae0d13b169897fc8a247d75fb5beadd69c5f0
 
 build:
-    number: 1
-    skip: True  # [py2k]
-    script: python -m pip install --no-deps --ignore-installed .
+    number: 2
+    noarch: python
+    script: python -m pip install --no-deps .
```
Comment (mbargull) [inline recipe/meta.yaml]: Using `--ignore-installed` should be fine, no need to remove it :wink:

Comment (blink1073) [reply recipe/meta.yaml]: Ha, that was me cargo-culting from a recipe that was working...

**Comment by blink1073** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 3 above)
Comment (mbargull) [reply recipe/meta.yaml]: Haha thanks, "cargo-culting", yet another new phrase I never heard before -- I'll try to remember it :laughing:

Comment (blink1073) [reply recipe/meta.yaml]: Ha, thanks for the review!

**Pull request merged**